integration_test/third_party_apps_data/applications/mongodb/metadata.yaml

```diff
@@ -81,6 +81,61 @@ expected_metrics:
   monitored_resource: gce_instance
   labels:
     database: .*
+- type: workload.googleapis.com/mongodb.extent.count
```

Comment:
I don't see this metric in the scope doc, does it need to be added there?

Reply:
Yeah, this should not be in this enhancements PR. It is in the scope doc but got left out of the original implementation. `extent.count`  can only be used for mongodb version <= 4.2  and if the storage engine is MMAPv1. The [default storage](https://www.mongodb.com/docs/manual/core/storage-engines/#storage-engines) engine as of starting with 3.2 is the Wired Tiger.  Updated.